GVFS/GVFS.UnitTests/Git/GitAuthenticationTests.cs

```diff
@@ -173,6 +175,7 @@ private MockGitProcess GetGitProcess()
             MockGitProcess gitProcess = new MockGitProcess();
             gitProcess.SetExpectedCommandResult("config gvfs.FunctionalTests.UserName", () => new GitProcess.Result(string.Empty, string.Empty, GitProcess.Result.GenericFailureCode));
             gitProcess.SetExpectedCommandResult("config gvfs.FunctionalTests.Password", () => new GitProcess.Result(string.Empty, string.Empty, GitProcess.Result.GenericFailureCode));
+            gitProcess.SetExpectedCommandResult("config --get-urlmatch http mock://repoUrl", () => new GitProcess.Result($"http.sslCert {CertificatePath}\nhttp.sslCertPasswordProtected true\n\n", string.Empty, GitProcess.Result.SuccessCode));

```

Comment:
It looks like with this change we won't get any coverage for the scenario where `http.sslCert` is not in the configuration.

Could you parameterize these tests so that we can get coverage for both scenarios? (see `DiffHelperTests` for an example of where we've parameterized unit tests in the past).